saleor/graphql/product/mutations/product_variant/product_variant_create.py

```diff
@@ -294,9 +296,20 @@ def check_for_duplicates_in_stocks(cls, stocks_data):
                 }
             )
 
+    @classmethod
+    def set_track_inventory(cls, _info, instance, cleaned_input):
+        track_inventory_by_default = get_track_inventory_by_default(_info)
+        if track_inventory_by_default is not None:
+            instance.track_inventory = (
+                track_inventory_by_default
+                if cleaned_input.get("track_inventory") is None
+                else cleaned_input.get("track_inventory")
+            )
```

Comment:
Same here:
```suggestion
                track_inventory = variant_data["cleaned_input"].get("track_inventory")
                variant.track_inventory = track_inventory or track_inventory_by_default
```

Reply:
Oh, right. The `track_inventory` is boolean. So:
```suggestion
            track_inventory = variant_data["cleaned_input"].get("track_inventory")
            variant.track_inventory = (
                track_inventory_by_default if track_inventory is None
                else track_inventory_by_default
            )
```

Reply:
@IKarbowiak I think then there is no change needed we can directly use the `cleaned_input.get("track_inventory")` instead of storing it on `track_inventory`.

Reply:
Yep, but you are doing it twice, and IMO it easier to read in a way I proposed. But it's just a suggestion. 🙂